compiler/src/main/kotlin/dev/zacsweers/metro/compiler/ir/Binding.kt

```diff
@@ -463,11 +465,20 @@ internal sealed interface Binding : BaseBinding<IrType, IrTypeKey, IrContextualT
         multibinds: IrAnnotation,
         contextualTypeKey: IrContextualTypeKey,
       ): Multibinding {
+        val isDaggerMultibinds =
+          multibinds.ir.symbol.owner.parentClassId?.asSingleFqName() ==
```

Comment:
Probably there is a better way to resolve classId or FqName

Reply:
cleaned up in c9feb68d2847d06c12c30da31ea785d1236746b2